ts/main.ts

```diff
@@ -45,11 +121,43 @@ const $collisionDiv = document.querySelector(
 const $closeMenuBtn = document.querySelector(
   '#close-menu-btn',
 ) as HTMLButtonElement;
+const $menuAllSpellsAnchor = document.querySelector(
+  '#menu-all-spells-anchor',
+) as HTMLAnchorElement;
+const $menuNewSpellbookAnchor = document.querySelector(
+  '#menu-new-spellbook-anchor',
+) as HTMLButtonElement;
+const $menuSpellbooksDiv = document.querySelector(
+  '#menu-spellbooks-div',
+) as HTMLDivElement;
 
 if (!$menuBtn) throw new Error('$menuBtn query failed');
 if (!$menuDialog) throw new Error('$menuDialog query failed');
 if (!$collisionDiv) throw new Error('$collisionDiv query failed');
 if (!$closeMenuBtn) throw new Error('$closeMenuBtn query failed');
+if (!$menuAllSpellsAnchor) throw new Error('$menuAllSpellsAnchor query failed');
+if (!$menuNewSpellbookAnchor)
+  throw new Error('$menuNewSpellbookAnchor query failed');
+if (!$menuSpellbooksDiv) throw new Error('$menuSpellbooksDiv query failed');
+
+function renderSpellbookLink(bookName: string, bookId: number): HTMLDivElement {
+  const $div = document.createElement('div');
+  $div.className = 'spellbook-link-div';
+
+  const $anchor = document.createElement('a');
+  $anchor.className = 'spellbook-link';
+  $anchor.textContent = bookName;
+  $anchor.setAttribute('data-id', bookId.toString());
+
+  $div.appendChild($anchor);
+
+  return $div;
+}
+
+spellbookData.spellbooks.forEach((element) => {
+  const $bookLink = renderSpellbookLink(element.name, element.id);
+  $menuSpellbooksDiv.appendChild($bookLink);
+});
```

Comment:
If this is a list of spell books, I would make these into an unordered list and then each spell book should be inside of a list item element.